build/build-snaps.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+set -e
+
+if [ $1 == "agent" -o $1 == "sdk" ]
+then
+    echo "Building $SNAP snap..."
+
+    if [ ! -d "/snap" ]
```

Comment:
"/snap" is the root snap folder. Change to "./snap" (also in the mkdir call for consistency).